interfaces/PA-App/src/app/services/user-ims-api.service.ts

```diff
@@ -144,4 +144,26 @@ export class UserImsApiService {
       )
       .toPromise();
   }
+
+  deleteWallet(
+    wallet: Wallet,
+  ): Promise<any> {
+    console.log('UserImsApiService : storeCredential()');
+
+    return this.apiService
+      .post(
+        environment.url_user_ims_api,
+        '/wallet/backup', // TODO change this to delete wallet endpoint when it's finished
+        {
+          wallet,
+          backupFileStoragePath: '/tmp/sdf', // TODO remove this after changing endpoint
```

Comment:
Does it need to be committed when its not finished?

Reply:
I dunno.. In the other project i'm working on they leave a lot of these #TODO placeholders scattered amongst the code